src/main/java/qlik/jdbc/connector/ConnectorImpl.java

```diff
@@ -55,6 +55,9 @@ public void getData(DataRequest request, StreamObserver<DataChunk> responseObser
             conn = DriverManager.getConnection(connectionString, request.getConnection().getUser(), request.getConnection().getPassword());
 
             stmt = conn.createStatement();
+            conn.setAutoCommit(false);
+
+            stmt.setFetchSize(100000);

```

Comment:
This value should probably be configurable. Maybe using an env variable and default to `100000` if not set?